vsts-ci.yml

```diff
@@ -41,6 +41,7 @@ steps:
     command: 'test'
     projects: 'VSConfigFinder.Test'
     arguments: '--configuration $(BuildConfiguration)'
+    publishWebProjects: false
```

Comment:
Isn't this the wrong task? Don't you want this on the publish task?

Reply:
Haha you're right! My mistake